appveyor.yml

```diff
@@ -73,9 +73,19 @@ install:
 
     - cmd: conda config --set show_channel_urls true
     - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
+    - cmd: conda install -n root --yes --quiet jinja2 conda-build anaconda-client=1.5.1
     - cmd: conda config --add channels conda-forge
-    - cmd: conda install -n root --quiet --yes conda-forge-build-setup
-    - cmd: run_conda_forge_build_setup
+    - cmd: set CPU_COUNT=2
```

Comment:
I just copied the whole file content from https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_win.bat here. No idea what this is for.